src/main/kotlin/at/raven/ravenAddons/commands/RefillCommand.kt

```diff
@@ -9,85 +10,67 @@ import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 @LoadModule
 object RefillCommand {
-    @SubscribeEvent
-    fun onCommandRegisteration(event: CommandRegistrationEvent) {
-        event.register("pearl") {
-            description = "Refill your stack of Ender Pearls to 16."
-            aliases = listOf("ep")
-            category = CommandCategory.REFILL
-            callback { refill("Ender Pearl", "ender_pearl", 16, "§f") }
-        }
-
-        event.register("jerry") {
-            description = "Refill your stack of Inflatable Jerrys to 64."
-            aliases = listOf("ij")
-            category = CommandCategory.REFILL
-            callback { refill("Inflatable Jerry", "inflatable_jerry", 64, "§f") }
-        }
-
-        event.register("superboom") {
-            description = "Refill your stack of Superboom TNT to 64."
-            aliases = listOf("sb")
-            category = CommandCategory.REFILL
-            callback { refill("Superboom TNT", "superboom_tnt", 64, "§9") }
-        }
-
-        event.register("leaps") {
-            description = "Refill your stack of Spirit Leaps to 16."
-            aliases = listOf("sl")
-            category = CommandCategory.REFILL
-            callback { refill("Spirit Leap", "spirit_leap", 16, "§9") }
-        }
-
-        event.register("decoy") {
-            description = "Refill your stack of Decoys to 64."
-            aliases = listOf("de")
-            category = CommandCategory.REFILL
-            callback { refill("Decoy", "decoy", 64, "§a") }
-        }
+    private enum class items(
+        val command: String,
+        val aliases: List<String>,
+        val description: String,
+        val item: String,
+        val id: String,
+        val stack: Int,
+        val colour: String,
```

Comment:
```suggestion
        val colour: String = "§f"
```

Can get rid of some repetitive stuff here